src/phoenix/server/api/helpers/dataset_helpers.py

```diff
@@ -79,10 +83,10 @@ def _get_llm_span_input(
     The input is extracted from the input messages (if present) and prompt
     template variables (if present).
     """
-    input: Dict[str, Any]
-    if input_messages is not None:
-        input = {"input_messages": input_messages}
-    else:
+    input: Dict[str, Any] = {}
+    if messages := [_get_message(m) for m in input_messages or ()]:
+        input["messages"] = messages
+    if not input:
```

Comment:
I don't think we want to grab the generic input if we already have the messages.

```suggestion
    else:
```

Reply:
nvm, it's equivalent.